truffle.js

```diff
@@ -14,20 +14,20 @@ module.exports = {
       network_id: '*'
     },
     live: {
-      provider: () => new HDWalletProvider(MNEMONIC, `https://mainnet.infura.io/${INFURA_TOKEN}`),
+      provider: () => new HDWalletProvider(MNEMONIC, `https://mainnet.infura.io/v3/${INFURA_TOKEN}`),
       network_id: '1',
     },
     kovan: {
-      provider: () => new HDWalletProvider(MNEMONIC, `https://kovan.infura.io/${INFURA_TOKEN}`),
+      provider: () => new HDWalletProvider(MNEMONIC, `https://kovan.infura.io/v3/${INFURA_TOKEN}`),
       network_id: '2',
     },
     ropsten: {
-      provider: () => new HDWalletProvider(MNEMONIC, `https://ropsten.infura.io/${INFURA_TOKEN}`),
+      provider: () => new HDWalletProvider(MNEMONIC, `https://ropsten.infura.io/v3/${INFURA_TOKEN}`),
       network_id: '3',
       gas: 4700000,
     },
     rinkeby: {
-      provider: () => new HDWalletProvider(MNEMONIC, `https://rinkeby.infura.io/${INFURA_TOKEN}`),
+      provider: () => new HDWalletProvider(MNEMONIC, `https://rinkeby.infura.io/v3/${INFURA_TOKEN}`),
```

Comment:
We provide this token in other repos correct? I think best to leave this as it was before so we don't go needing to change it elsewhere

Reply:
That is not an option. Infura has upgraded their API to v3, and without that addition, the link won't work. The others, if not changed, won't work either when deploying.

Reply:
Ahh, and the token remains the same there?  That sounds good then